etc/servo_automation_screenshot.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2018 The Servo Project Developers. See the COPYRIGHT
+# file at the top-level directory of this distribution.
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+"""
+Created on Mon Mar 26 20:08:25 2018
+@author: Pranshu Sinha, Abhay Soni, Aayushi Agrawal
+"""
+
+"""
+The below program is intended to test rendering mismatches in servo by taking screenshots of rendered html files.
+Here is the breakdown of how our code works:
+*   A session is started on localhost:7002
+*   The randomly generated webpage's (html files) data is sent as JSON to this session
+*   Using curl request, we load the html files for this session ID based on the session we just created.
+"""
+import os
+import json
+import requests
+import start_servo
+import time
+import base64
+import sys
+import getopt
+
+
+def servo_ready_to_accept(url, payload, headers):
+    while(True):
+
+        try:
+            # Before sending an additional request, we wait for one second each time
+            time.sleep(1)
+            session_request = requests.post(url, data=payload, headers=headers)
+            json_string = session_request.json()
+            # On success, we move on to render the files
+            break
+        except Exception as e:
+            time.sleep(5)
+            print('Exception: ', e)
+    return json_string
+
+
+def ensure_screenshots_directory_exists():
+    if not os.path.exists('screenshots'):
+        os.makedirs('screenshots')
+
+
+def render_html_files(num_of_files, url, file_url, json_string, headers, cwd):
+    for x in range(num_of_files):
+
+        json_data = {}
+        json_data['url'] = 'file://{0}file{1}.html'.format(file_url, str(x))
+        print(json_data['url'])
+        json_data = json.dumps(json_data)
+        requests.post('{0}/{1}/url'.format(url, json_string['value']['sessionId']), data=json_data, headers=headers)
```

Comment:
Leave some comments with links to webdriver docs about these endpoints and what's going on